orchagent/portsorch.cpp

```diff
@@ -6137,7 +6173,6 @@ bool PortsOrch::initializePort(Port &port)
         initializePriorityGroups(port);
         initializeQueues(port);
         initializeSchedulerGroups(port);
-        initializePortBufferMaximumParameters(port);
```

Comment:
Why is this change made to move init for buffer parms?

Reply:
@vaibhavhd This function sets max_headroom_size in STATE_DB for use by buffermgrd for buffer calculations. In fast boot, buffer settings already present in APPL_DB/CONFIG_DB. Also, no code uses max_headroom_size in orchagent.